tests/std/tests/GH_000431_lex_compare_memcmp_classify/test.compile.pass.cpp

```diff
@@ -252,8 +256,8 @@ void lex_compare_memcmp_classify_test_cases() {
     test_lex_compare_memcmp_classify_for_1byte_integrals<true, bool, char8_t>();
     test_lex_compare_memcmp_classify_for_1byte_integrals<true, char8_t, bool>();
 #endif // __cpp_lib_char8_t
-    test_lex_compare_memcmp_classify_for_1byte_integrals<false, char, bool>();
-    test_lex_compare_memcmp_classify_for_1byte_integrals<false, bool, char>();
+    test_lex_compare_memcmp_classify_for_1byte_integrals<vec_alg && is_unsigned_v<char>, char, bool>();
+    test_lex_compare_memcmp_classify_for_1byte_integrals<vec_alg && is_unsigned_v<char>, bool, char>();

```

Comment:
These two cases are different from others in that they were not optimized with `memcmp` though I expect they should have been.

Reply:
The reason why is that we barely care about `/J` which is a super weird and bad compiler option. (I manually verified that this test passes with it, but I don't think we need to add automated coverage.)